src/anomalib/metrics/per_image/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""Per-Image Metrics.
+
+author: jpcbertoldo
+"""
+
+# Original Code
+# Copyright (c) 2024 @jpcbertoldo
+# https://github.com/jpcbertoldo/aupimo
+# SPDX-License-Identifier: MIT
+#
+# Modified
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
this is a good idea, but we might need to double check with GSoC guidelines in regards to who owns the code.